fixtures/coverall/tests/bindings/test_coverall.kts

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import uniffi.coverall.*
+
+// Test some_dict().
```

Comment:
FYI, these were copied directly from the python test of the same name: https://github.com/mozilla/uniffi-rs/blob/e6bc3ef633929cba65250f36a50c4cb997bbc128/fixtures/coverall/tests/bindings/test_coverall.py#L9